test/glimesh/api/accounts_test.exs

```diff
@@ -138,6 +140,38 @@ defmodule Glimesh.Api.AccountsTest do
   }
   """
 
+  @follow_create_mutation """
+  mutation FollowCreate($channelId: ID!) {
+    followCreate(channelId: $channelId) {
```

Comment:
I changed how these mutations are named so they group together by the `follow` prefix. My prior experience with GraphQL is slowly coming back to me, this pattern of using prefixes to group related mutations is considered best practice as there is no other good way to indicate related mutations. It's not a perfect solution, but I think it's better than having `follow`, `unfollow`, `updateFollow` mutations.

Reference https://github.com/Shopify/graphql-design-tutorial/blob/master/TUTORIAL.md#naming-the-mutations

Reply:
Sorry for the delay, I understand the thought behind naming them this way, but it feels a bit counter intuitive given how verb based the rest of the API is `watchChannel`, `logStreamMetadata`, `uploadStreamThumbnail`. Instead of thinking abstractly in "fields that are updated" I like the API to be thought of as "actions that are preformed".

In this case, I propose having two mutations: 
* `follow(channelId: ID!, hasLiveNotifications: Boolean = false)` - Which is used for both following someone initially, and updating a follow to use live notifications or not. 
* `unfollow(channelId: ID!)` - Which is used to unfollow a channel.

Reply:
Let me know what you think, everything else in the PR is good with me!

Reply:
Thanks, 👍 perfectly happy sticking to verbs, updated the PR.

Agreed it's good to think of mutations as verbs, I try to do that also but just had a flashback to dealing with some very large APIs where that gets messy. Not so relevant with <20 mutations haha

There is only one edge case, and it's unlikely to ever come up. Calling `follow` and not specifying a value for `liveNotifications` course defaults to `false` for a new follow. The edge case is when there is an existing follow. I decided to have it change the value in the database to `false` in that case, vs just leaving it alone.

Reply:
Sounds good! Thanks for the work, one last review and I'll merge / deploy this morning.